versioned_docs/version-3.0/releasenotes/release-3.0.0.md

```diff
@@ -4,3 +4,166 @@
+
+
+Production Insights
+
+**1. Operations-Friendly**
+
+- Cache TTL: In real-time scenarios, setting the Cache TTL ensures that data within the TTL provides low-latency queries from the local cache.
+
+- Seamless Scaling: One benefit of the compute-storage decoupled mode is the rapid elasticity of computing resources. Doris uses progressive active cache preheating technology, ensuring no impact on ongoing queries and imports during scalability periods.
+
+- Seamless Upgrades: During upgrades, the multi-process mode operation ensures ongoing queries and imports do not fail.
+
+
+
+**2. Load Isolation**
+
+- Multiple Computing Clusters: Different business characteristics (e.g., batch processing, real-time) or business units (different businesses within a group) can use independent computing clusters for physical isolation.
+
+- Read-Write Separation: Imports and queries can be completed by different computing clusters, supporting automatic (notification for preheating during import) and manual (periodic manual preheating) methods to preheat the query cluster's cache.
+
+
+
+**3. Positive Data Deletion**
+
+In distributed systems, common data deletion involves reverse GC, comparing files in shared storage with those recorded in metadata and deleting unrecorded files. This method can easily lead to data deletion errors under concurrent conditions. Relying on MetaService's transaction capabilities, Doris adopts asynchronous forward data deletion technology. Data generation is bound to a transaction, as is metadata deletion, and the asynchronous deletion process simply deletes data based on transaction records, eliminating the need for reverse GC.
+
+
+
+## Optimizer
+
+- All DQL and DML, except for load operations, will use the new optimizer. Default settings disable fallback to the old optimizer, which will be removed in subsequent versions.
```

Comment:
Suggestion: Elaborate on what the new optimizer is / link to docs to learn more.